builder-support/helpers/install_quiche.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/sh
+set -v
+set -e
+
+readonly QUICHE_VERSION='0.18.0'
+readonly QUICHE_TARBALL="${QUICHE_VERSION}.tar.gz"
+readonly QUICHE_TARBALL_URL="https://github.com/cloudflare/quiche/archive/${QUICHE_TARBALL}"
+readonly QUICHE_TARBALL_HASH='eb242a14c4d801a90b57b6021dd29f7a62099f3a4d7a7ba889e105f8328e6c1f'
+
+cd /tmp
+echo $0: Downloading $QUICHE_TARBALL
+curl -L -o "${QUICHE_TARBALL}" "${QUICHE_TARBALL_URL}"
+# Line below should echo two spaces between digest and name
+echo "${QUICHE_TARBALL_HASH}"  "${QUICHE_TARBALL}" | sha256sum -c -
+tar xf "${QUICHE_TARBALL}"
```

Comment:
`sha256sum` is not a command that iis available on OpenBSD. But as the embeddded boringSSL does not build on OpenBSD, the issue is somewhat mood.

Reply:
Do you happen to know if BoringSSL (not embedded) builds on OpenBSD? Because we do not have to use the embedded version so it might be a solution.

Reply:
```
[otto@lou:2]$ doas pkg_add boringssl 
doas (otto@lou.intra.drijf.net) password: 
quirks-6.157 signed on 2023-09-28T08:57:09Z
boringssl-20230824: ok
```

Reply:
Thanks! So we can very likely fix this issue by telling Quiche to use the system BoringSSL, although thinking about it a bit more, I'm worried this would cause a conflict later when Quiche tries to get the symbols from BoringSSL while OpenSSL is also loaded. This avoids this issue when BoringSSL is part of `libquiche` because then the relocation is done inside the same shared library.